cfgov/paying_for_college/disclosures/scripts/update_colleges.py

```diff
@@ -60,8 +60,8 @@ def update_programs(api_data, school):
                 'program_name': entry['title'],
                 'cip_code': cip_code,
                 'completers': entry['counts']['titleiv'],
-                'level_code': level_code,
-                'level': PROGRAM_LEVELS.get(int(level_code)),
+                'level': level_code,
```

Comment:
So `level` became `level_name`, and `level_code` became `level`? Is there some data migration needed, or is this not needed because this data isn't live yet?

Reply:
Data collection evolved in a confusing way here, but there's no data migration needed.  

The following is surely TMI, but here goes:  

The name swapping is driven by the need to keep our API backward-compatible for the original tool, which used program data provided by schools, while dealing with a new source for program data -- CollegeScorecard.

#### The evolution 
- The former db column `level_code` was intended to reflect a program's numeric level, but was never populated or used. In our data [from schools](https://cfpb.github.io/college-costs/csv-spec/), the numeric "program_level" CSV value got stored as the db value "Program.level" and was delivered via our API to the tool as `level`.
- Now that program data is available from CollegeSorecard, as opposed to getting it directly from schools, we want to have the level's name available in our API, while leaving our API var `level` untouched. So we're renaming the empty db column "level_code" to "level_name" for that purpose. 
- The db column "level" is left alone, and we map CollegeScorecard's numerical `level` value  to that. The `update_iped` script's use of the var `level_code` was contributing to the confusion, so I renamed it `level` to match Scorecard and our db value.